packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -118,3 +118,23 @@ export const allocate = async (metricsAddress: string) => {
   }
   return undefined
 }
+
+export const createProperty = async (name: string, symbol: string, author: string) => {
+  const client = newClient()
+  if (process.env.NODE_ENV == 'production' && client) {
+    return client
+      .propertyFactory(await client.registry(addresses.eth.main.registry).propertyFactory())
+      .createProperty(name, symbol, author)
+  } else if (process.env.NODE_ENV == 'development') {
+    console.log('env:', process.env.NODE_ENV, 'return mock value')
+  }
+  return undefined
```

Comment:
A simple mock of the Create property is defined by checking the environment variables.

Reply:
I understood. I configured the value of `NODE_ENV` on Vercel. In the production, it's `production`, and other than it, it's `development`.

Reply:
Thank you for adjusting the development environment!